interp/interp_test.go

```diff
@@ -57,16 +57,14 @@ func runTest(t *testing.T, pathPrefix string) {
 	if err != nil {
 		if err, match := err.(*Error); match {
 			println(err.Error())
-			if !err.Inst.IsNil() {
-				err.Inst.Dump()
-				println()
+			if len(err.Inst) != 0 {
+				println(err.Inst)
```

Comment:
Oh and these need to be changed to `t.Log` or something eventually (I think the reason I used println before was so the error messages would interleave in a sensible way with `inst.Dump()`, which I think also writes to stderr).